### PR TITLE
[PAY-1514] add UK/GBR as supported country in reimbursement offer automation

### PR DESCRIPTION
**Pull request by pedroalmeida-fidel**
[Deploy preview](https://pr-825.d30jf5gj5vctl9.amplifyapp.com/docs/reimbursement)

### CODE REVIEW

File: reimbursement.md [hunk 1]
```diff
@@ -4,17 +4,16 @@ Fidel Reimbursement is an add-on product to _Transaction_ tracking capability, t
 
 ## Availability
 
-Reimbursement supports both debit and credits cards by Visa and Mastercard in the US. Your account must be setup in the US to have access to the Reimbursement product.
-
+Reimbursement supports both debit and credit cards by Visa and Mastercard in the US or UK. Your account must be set up either in the US or the UK to have access to the Reimbursement product.
 ## Activation
 
-If your account is setup in the US and you are using an older [API version](https://reference.fidel.uk/docs/version-log) (reimbursement requires version `2021-09-28` or later) or haven't agreed to the _North America_ [Terms & Conditions](https://fidel.uk/legal) (_17 Jun 2021_), you will be prompted to go through the Reimbursement product activation.
+If your account is set up in the US or UK and you are using an older API version (reimbursement requires version 2021-09-28 or later) or haven't agreed to the (Terms & Conditions)[Link to T&Cs page] in your region, you will be prompted to go through the Reimbursement product activation.
```
Comment (janos-fidel) [inline reimbursement.md:10]:
```suggestion
If your account is set up in the US or UK and you are using an older API version (reimbursement requires version 2021-09-28 or later) or haven't agreed to the [Terms & Conditions](https://fidel.uk/legal) in your region, you will be prompted to go through the Reimbursement product activation.
```

**Review by janos-fidel — APPROVED** [review]

**Review by krisztina-nagy — APPROVED** [review]

**Review by krisztina-nagy — DISMISSED** [review]

**Review by PedroMGSilva — DISMISSED** [review]

File: reimbursement.md [hunk 2]
```diff
@@ -4,17 +4,16 @@ Fidel Reimbursement is an add-on product to _Transaction_ tracking capability, t
 
 ## Availability
 
-Reimbursement supports both debit and credit cards by Visa and Mastercard in the US. Your account must be set up in the US to have access to the Reimbursement product.
-
+Reimbursement supports both debit and credit cards by Visa and Mastercard in the US or UK. Your account must be set up either in the US or the UK to have access to the Reimbursement product.
 ## Activation
 
-If your account is set up in the US and you are using an older [API version](https://reference.fidel.uk/docs/version-log) (reimbursement requires version `2021-09-28` or later) or haven't agreed to the _North America_ [Terms & Conditions](https://fidel.uk/legal) (_17 Jun 2021_), you will be prompted to go through the Reimbursement product activation.
+If your account is set up in the US or UK and you are using an older API version (reimbursement requires version 2021-09-28 or later) or haven't agreed to the [Terms & Conditions](https://fidel.uk/legal) in your region, you will be prompted to go through the Reimbursement product activation.
```
Comment (janos-fidel) [inline reimbursement.md:10]: Shouldn't we use the new URL for the T&C link? `https://fidelapi.com/legal`

**Review by janos-fidel — DISMISSED** [review]

File: reimbursement.md (hunk 2 above)
Comment (janos-fidel) [reply reimbursement.md:10]: Unrelated to this PR, but do we need to update the T&Cs on the legal page?

Comment (pedroalmeida-fidel) [reply reimbursement.md]: you're right, i'll update this

Comment (pedroalmeida-fidel) [reply reimbursement.md]: that's a good point - we can bring this into Ruben's attention

**Review by LucasSindra — APPROVED** [review]

File: reimbursement.md [hunk 3]
```diff
@@ -26,7 +25,7 @@ Your credit balance is updated every time you purchase credits or spend by using
 
 ### Credits Balance Example
 
-Balances are denominated in multiple currencies to support other countries in the future. Currently we support `USD` and in this example `balances.USD` value is `$1000`.
+Balances are denominated in multiple currencies to support other countries in the future. Currently, we support either `USD` or `GBP` and in this example `balances.USD` value is `$1000`.
```
Comment (LucasSindra) [inline reimbursement.md:28]: Couldn't we use 'both' instead of 'either'?

File: reimbursement.md [hunk 4]
```diff
@@ -373,7 +372,7 @@ For more info, please visit the [API reference](https://reference.fidel.uk/refer
 
 ## Automation
 
-Automate reimbursement requests when creating an Offer by toggling the “enable automatic reimbursements” checkbox. This toggle will be available if the reimbursement product is active and the Offer’s country is the `USA`.
+Automate reimbursement requests when creating an Offer by toggling the “enable automatic reimbursements” checkbox. This toggle will be available if the reimbursement product is active and the Offer’s country is either `USA` (United States of America) or `GBR` (United Kingdom).
```
Comment (LucasSindra) [inline reimbursement.md:375]: Couldn't we use 'both' instead of 'either'?

File: reimbursement.md (hunk 3 above)
Comment (pedroalmeida-fidel) [reply reimbursement.md:28]: It's an option @LucasSindra 
@krisztina-nagy, what's your take on this?

**Review by janos-fidel — DISMISSED** [review]

File: reimbursement.md [hunk 5]
```diff
@@ -210,7 +209,7 @@ fileName:transaction.json
 
 ### Creating a request
 
-After choosing the `transactionId`, the reimbursement `amount` must be equal to or lower than the transaction `amount` and the `currency` is determined by the transaction. Visa cards have a maximum reimbursement amount limit of _USD $250_. We currently support `USD` currency transactions. Optionally customise the `description` text that will show in the cardholder bank statement. Read more information on the [create reimbursement endpoint](https://reference.fidel.uk/reference/create-reimbursement).
+After choosing the `transactionId`, the reimbursement `amount` must be equal to or lower than the transaction `amount` and the `currency` is determined by the transaction. Visa cards have a maximum reimbursement amount limit of _USD $250_. We currently support `USD` or `GBP` currency transactions. Optionally customise the `description` text that will show in the cardholder bank statement. Read more information on the [create reimbursement endpoint](https://reference.fidel.uk/reference/create-reimbursement).
```
Comment (LucasSindra) [inline reimbursement.md:212]: Maybe this "or" could be an "and" too

**Review by PedroMGSilva — APPROVED** [review]

**Review by krisztina-nagy — APPROVED** [review]

**Review by janos-fidel — DISMISSED** [review]

**Review by LucasSindra — APPROVED** [review]

**Review by janos-fidel — APPROVED** [review]

**Pull request merged**